definitions/private-law/json/AuthorisationCaseField/ServiceOfApplication/AuthorisationCaseField.json

```diff
@@ -1118,5 +1118,130 @@
+        },
+        {
+          "UserRoles": [
+            "citizen",
+            "caseworker-privatelaw-solicitor",
+            "caseworker-privatelaw-superuser",
+            "caseworker-wa-task-configuration",
+            "caseworker-privatelaw-readonly"
+          ],
+          "CRUD": "R"
+        }
+      ]
+    },
+    {
+      "LiveFrom": "09/08/2023",
+      "CaseTypeID": "PRLAPPS",
+      "CaseFieldID": "soaServeC8ToLocalAuthorityYesOrNo",
+      "AccessControl": [
+        {
+          "UserRoles": [
+            "caseworker-privatelaw-la",
+            "caseworker-privatelaw-judge",
+            "caseworker-privatelaw-courtadmin",
+            "caseworker-privatelaw-systemupdate"
+          ],
+          "CRUD": "CRUD"
+        },
+        {
+          "UserRoles": [
+            "citizen",
+            "caseworker-privatelaw-solicitor",
+            "caseworker-privatelaw-superuser",
+            "caseworker-wa-task-configuration",
+            "caseworker-privatelaw-readonly"
+          ],
+          "CRUD": "R"
+        }
+      ]
+    },
+    {
+      "LiveFrom": "09/08/2023",
+      "CaseTypeID": "PRLAPPS",
+      "CaseFieldID": "soaDocumentDynamicListForLa",
+      "AccessControl": [
+        {
+          "UserRoles": [
+            "caseworker-privatelaw-la",
+            "caseworker-privatelaw-judge",
+            "caseworker-privatelaw-courtadmin",
+            "caseworker-privatelaw-systemupdate"
+          ],
+          "CRUD": "CRUD"
+        },
+        {
+          "UserRoles": [
+            "citizen",
+            "caseworker-privatelaw-solicitor",
+            "caseworker-privatelaw-superuser",
+            "caseworker-wa-task-configuration",
+            "caseworker-privatelaw-readonly"
+          ],
+          "CRUD": "R"
+        }
+      ]
+    },
+    {
+      "LiveFrom": "09/08/2023",
+      "CaseTypeID": "PRLAPPS",
+      "CaseFieldID": "soaLaEmailAddress",
+      "AccessControl": [
+        {
+          "UserRoles": [
+            "caseworker-privatelaw-la",
+            "caseworker-privatelaw-judge",
+            "caseworker-privatelaw-courtadmin",
+            "caseworker-privatelaw-systemupdate"
+          ],
+          "CRUD": "CRUD"
+        },
+        {
+          "UserRoles": [
+            "citizen",
+            "caseworker-privatelaw-solicitor",
+            "caseworker-privatelaw-superuser",
+            "caseworker-wa-task-configuration",
+            "caseworker-privatelaw-readonly"
+          ],
+          "CRUD": "R"
+        }
+      ]
+    },
+    {
+      "LiveFrom": "09/08/2023",
+      "CaseTypeID": "PRLAPPS",
+      "CaseFieldID": "addDocumentsForLaLabel",
+      "AccessControl": [
+        {
+          "UserRoles": [
+            "caseworker-privatelaw-la",
+            "caseworker-privatelaw-judge",
+            "caseworker-privatelaw-courtadmin",
+            "caseworker-privatelaw-systemupdate",
+            "citizen",
+            "caseworker-privatelaw-solicitor",
```

Comment:
can this be changed ? compare CRUD permission for other fields, for ex: system-update should always have CRU